src/label.jsx

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+import React from 'react';
+import classnames from 'classnames';
+
+export var Label = React.createClass({
+
+  render: function () {
+    var template, className, label;
```

Comment:
use `let` for `template` & `const` for the other two. Also declare where they are used.